packages/aws-cdk/lib/init-templates/app/dotnet/src/HelloCdk/HelloConstruct.cs

```diff
@@ -0,0 +1,28 @@
+using Amazon.CDK;
+using Amazon.CDK.AWS.IAM;
+using Amazon.CDK.AWS.S3;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace HelloCdk
+{
+    public class HelloConstruct : Construct
```

Comment:
Add some documentation across the board to explain what you are doing

Reply:
✔️ 
FYI, the Java template has no documentation either. Once this PR is finished, I'll create another for (near-identical) Java comments.